plugins/outputs/cloudwatch/util.go

```diff
@@ -50,19 +47,12 @@ const (
 )
 
 func publishJitter(publishInterval time.Duration) (publishJitter time.Duration) {
-	hostName, _ := os.Hostname()
-	jitter := computeMD5Hash(hostName, int64(publishInterval.Seconds()))
+	rand.Seed(time.Now().Unix())
```

Comment:
Using the package level `rand.Seed` changes the global default, which may have impact on some library we are using https://golang.org/pkg/math/rand/#Seed

Maybe we can create a package specific instance and use that instead?

```go
var r = rand.New() // can have a better longer name ..

func init() {
     r.Seed(time.Now().Unix())
}
```